File: src/ERP/acme.atena.domain/DTO/Product/DevolucaoVenda.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace acme.atena.domain.DTO.Product
{
    public class DevolucaoVenda : AbstractEntityCompetencia
    {
        public string Motivo { get; set; }
        public Guid VendaProdutoId { get; set; }
        public Guid ProdutoId { get; set; }
        public bool IsParcial { get; set; } = false;

        public virtual VendaProduto VendaProduto { get; set; }
        public virtual Produto Produto { get; set; }

    }
}

File: site/coreui-free-angular-admin-template-master/src/app/view-model/response/product/devolucao-venda.ts
import { Produto } from './produto';
import { VendaProduto } from './venda-produto';
import { AbstractEntityCompetencia } from "../abstract-entity-competencia";

export class DevolucaoVenda extends AbstractEntityCompetencia{
  public motivo:string;
  public vendaProdutoId: string;
  public produtoId: string;
  public isParcial :boolean;

  public vendaProduto: VendaProduto;
  public produto: Produto;

}
